css/style.css

```diff
@@ -0,0 +1,68 @@
+/*................................................
+    HEADER SECTION
+.................................................*/
+
+.container {
+    max-width: 1600px;
```

Comment:
do not use arbitrary values for width . 
Fix this   - apply a reset css  to `html` and `body` tags
Also use percentage values for width  when styles will get affected on resize